src/GuestBook/VIPGuest.java

```diff
@@ -0,0 +1,19 @@
+package GuestBook;
+
+public class VIPGuest implements Guest {
+	
+	private String name;
+	
+	public VIPGuest (String name) {
+		this.name = "VIP: " + name;
```

Comment:
The `name` instance variable should store the person's name, not their name with "VIP: " in front :)